### DISCUSSION TITLE
Rlink2 v2 to v3. Bluetooth call issue. No sound.

### DISCUSSION THREAD

**Discussion by Pikaa-IT** [cedricp/ddt4all#1092] [General]
Hi all.

Since I "pumped up" my Rlink2 from v 2.2.19.300 to v 3.3.16.966, Rlink2 has changed a few things, disabled.

Renault workshop does not help because there is foreign data in the system.

Perhaps someone is familiar with DDT4ALL, or Python

Problem 1: becoming phone calls is no sound. I have reset everything, but it does not help. Music works via Bluetooth. Calls do not.

I also tried with Python, didn't work either (maybe I did something wrong, or that macro is not for v3)

Problem 2: My Talisman is no longer Talisman 😶 but Scenic. See picture 1.
![Bild 1](https://github.com/cedricp/ddt4all/assets/168703540/70f52881-7aed-4946-bf29-82a400d4da16)

I can't change it with Ddt4all. Because of the red field I cannot change Vehicle Type. See picture 2 and 3.
![Bild 2](https://github.com/cedricp/ddt4all/assets/168703540/76a2d0c5-5834-4a42-b8af-7c6c0f143fac)
![Bild 3](https://github.com/cedricp/ddt4all/assets/168703540/cc19db50-18fc-47b4-9089-fb9a8e96aa5f)

Problem 3: Engine sound cannot be changed either. See video: https://photos.app.goo.gl/dBPNwFzMR3i8KujT9

Update is also not possible with Rlink2 Toolbox, because Toolbox does not recognize the current software version. It shows v2.
I don't know how the downgrade works and I don't have an older software version. I have not found any on the internet that I can download.

I hope someone can help me.

**Comment by matzel82** [comment]
Sorry, but that's exactly the reason why you shouldn't mess around with the RLink. I can understand why the workshop doesn't want to touch it.

**Reply by Pikaa-IT** [reply]
Yes, I know. I've learned it now. But it's already too late. Now I need a solution to fix it.

**Comment by matzel82** [comment]
Now you can only try to find the current version that is right for your vehicle or to do a downgrade and then an official update. I don't really have any other ideas. Otherwise you will actually need a new radio control unit that is right for your vehicle. Not the control unit. The control unit is located behind it.

**Reply by Pikaa-IT** [reply]
How can I perform the downgrade? What data do I need for this? Rlink2 says in the car that older software version is not installable.
